extra_data/reader.py

```diff
@@ -1993,26 +1994,35 @@ def open_run(
         the proposal directory path. By default it looks for a file named
         ``{}/usr/extra-data-aliases.yml``.
     """
+    absence_ok = set()
+    if data == 'default':
+        data = ['proc', 'raw']
+        absence_ok = {'proc'}
     if data == 'all':
```

Comment:
It's somewhat nitpicking, but an `elif` makes the flow clearer.

Reply:
Thanks, done!